molecule/shift-stack/converge.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: "execute deploy shift-stack"
+  import_playbook: ../../ansible/deploy_shift_stack.yaml
+  vars:
+    pull_secret_path: "../../../pull-secret"
```

Comment:
this default looks different to ``"~/.cache/ard/pull-secret"`` provided elsewhere

Reply:
yes i know i have been debating if i want to to take ti from the root of the directory or have you copy it to the cache location.

i have not added caching yet but when i did it will likely check both and only copy it form the root of the repo if not already present as a result of the caching.